test-three-nathanpeter/menu/src/MenuBarStyles.js

```diff
@@ -0,0 +1,51 @@
+import styled from "styled-components";
+
+export const MenuContainer = styled.div`
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+
+ 
+  text-align: center;
+  
+`;
+
+export const DropBtn = styled.div`
+  background-color: #2f76cc;
+  color: white;
+  padding: 16px;
+  font-size: 16px;
+  border: none;
+  cursor: pointer;
+
+`;
+
+export const DropdownContent = styled.div`
+  li {
```

Comment:
curious why you chose to use traditional css cascading here but use component cascading further down?

Reply:
its similar to using scss for me was just easier to do it this way rather than making a complete new li styled component